### PR TITLE
Input phrase

### LINKED ISSUE #307: Input Phrase steps in lessons
I've [created](https://github.com/braille-systems/learn-braille/commit/e635d64be665e3227220e758352e8ec7bc284a80) a UI template for a step where the user enters a word, phrase, or other combination of symbols.

- **The task is** to create classes `InputPhrase` and `InputPhraseFragment` containing current material, previous materials, and remaining next materials to display them in the top text pane (and set `top_text_pane`'s `android:contentDescription` to something like "enter phrase: ABC DEFG, already entered: ABC DE, remaining: FG").
Then replace `+Input` in `StepsBuilder.inputChars` to `+InputPhrase`.

I suppose that `InputPhraseFragment` could be quite reasonably inherited from `InputSymbolFragment` with minor changes in `onCreateView`, but AndroidStudio says that `InputSymbol` is a final class (likewise, I failed to inherit a class `InputPhrase` from `Input`). I prefer passing the task to @winter-yuki over delving further.

- Then there is still some work for @zuevval: add proper first and last step in a sequence, add help message, ensure that everything looks neat in terms of accessibility and course integrity.

---

This task is important because when it is done, it will be easier (I think) to
1. split lessons into "theoretic" lessons which consist mainly of non-input steps and "rehearsal" lessons which consist solely of `inputChars` (so we would be able to implement per-phrase navigation in these lessons)
1. add a special type of practice - typing in words

### PR DESCRIPTION
**Pull request by winter-yuki**
Allow users to input phrases symbol by symbol.
Closes #307

### CODE REVIEW

**Comment by winter-yuki** [comment]
I've begun to implement, but my Gradle plugin updated and I am unable to resolve building errors

**Comment by zuevval** [comment]
@winter-yuki we'll add migration a bit later so I suppose this request is ready for review. But it is unclear who should review it (you?)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -404,6 +408,10 @@
         ]]>
     </string>
 
+    <string name="lessons_help_input_phrase">
+        TODO
```
Comment (winter-yuki) [inline app/src/main/res/values/strings.xml:412]: Will it be added later?

**Review by winter-yuki — COMMENTED** [review]
Everything looks good

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (zuevval) [reply app/src/main/res/values/strings.xml:412]: Oops, thanks, I’ll add it tomorrow and then merge

Comment (zuevval) [reply app/src/main/res/values/strings.xml]: I decided to use the same help message as for `InputSymbol` fragments

**Pull request merged**